### PR TITLE
add mainnet and testnet configurations

### LINKED ISSUE #25: Problem: predefined network configuration
Add mainnet and testnet network configuration like `grpc_endpoint`, `chain_id`, `address_prefix` and etc\

This helps to prevent typo errors

### PR DESCRIPTION
**Pull request by macong-cdc**
Fix #25

### CODE REVIEW

**Comment by macong-cdc** [comment]
Please ignore lgtm-com alert message, it is created as an issue #30 already

**Comment by macong-cdc** [comment]
> This pull request **introduces 2 alerts** when merging [da077c9](https://github.com/crypto-org-chain/chainlibpy/commit/da077c93da0e9e435503e27e4f6117697ea0c60b) into [86f4e1f](https://github.com/crypto-org-chain/chainlibpy/commit/86f4e1feaa2bb24e9db3252767115899aa6b4650) - [view on LGTM.com](https://lgtm.com/projects/g/crypto-org-chain/chainlibpy/rev/pr-f7b89d1a6cb87790d5e953812eb650581fe44adc)
> 
> **new alerts:**
> 
> * 2 for Wrong number of arguments in a class instantiation

Please ignore lgtm-com alert message, it is created as an issue #30  already

File: chainlibpy/grpc_client.py [hunk 1]
```diff
@@ -44,28 +47,67 @@
...
     DEFAULT_GAS_LIMIT = 200000
 
     def __init__(
         self,
         wallet: Wallet,
-        chain_id: str,
-        grpc_endpoint: str,
+        network: NetworkConfig,
         credentials: ChannelCredentials = None,
     ) -> None:
```
Comment (linfeng-crypto) [inline chainlibpy/grpc_client.py:88]: the interface changed, and the version needs to be updated too I think.

Comment (macong-cdc) [reply chainlibpy/grpc_client.py:88]: Actually, the old version is not released yet.

File: tests/test_grpc_client.py [hunk 2]
```diff
@@ -1,73 +1,27 @@
...
-    add_auth(MockAuthQueryServicer(), _grpc_server)
-    add_tx(MockTxServiceServicer(), _grpc_server)
+@pytest.mark.parametrize("network_config", CRO_NETWORK.values())
+def test_network_config(network_config: "NetworkConfig"):
+    wallet = Wallet.new(path=network_config.derivation_path, hrp=network_config.address_prefix)
 
-    _grpc_server.add_insecure_port(grpc_addr)
-    _grpc_server.start()
+    server_host = network_config.grpc_endpoint.split(":")[0]
+    server_port = network_config.grpc_endpoint.split(":")[1]
```
Comment (linfeng-crypto) [inline tests/test_grpc_client.py]: `(server_host, server_port) = network_config.grpc_endpoint.split(":")`

**Pull request merged**